packages/fxa-auth-server/lib/routes/oauth/authorization.js

```diff
@@ -129,16 +129,26 @@ module.exports = ({ log, oauthDB, config }) => {
     // Clients must use a single specific redirect_uri,
     // but they're allowed to not provide one and have us fill it in automatically.
     payload.redirect_uri = payload.redirect_uri || client.redirectUri;
-    if (payload.redirect_uri !== client.redirectUri) {
-      log.debug('redirect.mismatch', {
-        param: payload.redirect_uri,
-        registered: client.redirectUri,
-      });
+
+    // Starting in train-248, FxA supports a comma separated list of redirect urls.
+    // However, the client redirectUri must be an exact match, pattern matching
+    // is not supported.
+    const redirectUris = client.redirectUri.split(',');
+
+    const validUri = redirectUris.some((uri) => {
+      if (uri === payload.redirect_uri) {
+        return true;
+      }
+      return false;
+    });
+
+    if (!validUri) {
       if (
         config.oauthServer.localRedirects &&
         isLocalHost(payload.redirect_uri)
       ) {
         log.debug('redirect.local', { uri: payload.redirect_uri });
+        return true;
```

Comment:
What's using this value? It doesn't look like we consistently return a boolean for this function.